content/en/getting_started/integrations/_index.md

```diff
@@ -134,6 +134,25 @@ Each integration has one of three status types:
 
 For information on how Datadog handles your data, and other security considerations, see the [Security documentation][30].
 
+## Granular Access Control
+By default, access to integration resources (accounts, services, webhooks) is unrestricted. Granular access controls can be used to restrict the behavior of users, teams, roles, or your full organization at the integration resource level. Not all integrations support granular access controls yet. 
```

Comment:
```suggestion
By default, access to integration resources (accounts, services, webhooks) is unrestricted. Granular access controls can be used to restrict the behavior of users, teams, roles, or your full organization at the integration resource level.
```

We generally list what we do support, but do not list what we do not support. With that said, is GRACE for integrations in Preview? We could add an alert box that says that.